pkg/kubelet/dockertools/manager.go

```diff
@@ -263,6 +273,18 @@ func (sc *reasonInfoCache) Get(uid types.UID, name string) (reasonInfo, bool) {
 	}
 }
 
+func (dm *DockerManager) isDockerAPIVersionCompatible(input string) bool {
+	version, err := dm.Version()
+	if err == nil {
```

Comment:
In go, we typically (although not always) check for a non-nil error and return early. So it should look something more like this:

```go
if err != nil {
  return false
}
if result, err := version.Compare(input); err == nil && result >= 0 {
  return true
}
return false
```